docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+
+.. image:: ../_static/control_center_pics/import_defaults_popup.png
+
+6. After clicking Yes, you will see your imported defaults as an option in the dropdown. This file will now always be available in the Control Center
+7. Once in the Control Center, you can use the Set button exactly as you would with a Vertiq supplied Defaults file
+
+Importing a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It is also possible to import externally generated Defaults files into the Control Center. To do so:
+
+#. Select the Import option
+#. Use the File Explorer to find the Defaults file you would like to import
+#. You will see the file appear in the dropdown of other Defaults files, and it is ready for use
+
+******************************************
+Testing Modules with the Control Center
+******************************************
+The Control Center provides a basic suite of testing options to ensure that your module is functioning properly. These options are available in the testing tab, 
+available on the left side.
+
+.. note::
+    Before using the Control Center's testing functionality, we recommend that you increase your module's timeout parameter to 1.5s. 
+    This parameter is available through the tuning tab. If you do not increase the timeout value, your module may fail to spin as expected.
+
+.. warning::
+    Please remove all propellers before interacting with any testing parameters. Failure to do so is dangerous.
+
+In order to set a testing parameter, simply enter a value into the spin box, and hit the down arrow set button. Parameters such as Coast do not have a spin box, 
+and only require that you press the set button to trigger the behavior.
+
+The testing tab for a Veritq 23-06 2200Kv looks as follows:
+
+.. image:: ../_static/control_center_pics/23_testing_tab.png
+    :scale: 50%
+
+Brake
+=========
+This commands the module to brake, which shorts the motor phases, which slows the motor down by dissipating energy in the motor.
+
+.. warning::
+    If you are using a power supply to power your module, it is possible to damage or destroy your module with aggressive commands, e.g. quickly switching from spinning at full speed to stopping. 
+    This is because Vertiq modules can also act as generators. In general, power supplies, unlike batteries, cannot absorb the energy generated by the module. As such, aggressive commands can lead to dangerous 
+    voltage spikes when connected to a power supply. To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the regeneration voltage protection feature.
+
+Coast
+=============
+Commanding a coast means that the module allows its rotor to slowly decelerate and come to a stop. It is a safe way to stop the module when testing since it does not cause the module to rapidly decelerate.
```

Comment:
Consider clarifying that it allows to slowly decelerate by disconnecting all the phases and letting it spin freely, its not like its intentionally driving it gradually to a stop. The wording now could be a bit ambiguous for someone not in the know